config/routes.rb

```diff
@@ -7,7 +7,7 @@
     mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
   end
 
-  devise_for :users, controllers: { registrations: 'registrations' }
+  devise_for :users, :controllers => { :registrations => 'registrations', :omniauth_callbacks => "callbacks" }
```

Comment:
Let's keep the syntax with the Ruby 2 style:

`controllers: { registrations: 'registrations', omniauth_callbacks: 'omniauth_callbacks' }`